mail/dovecot2/Portfile

```diff
@@ -44,9 +42,10 @@ patch.pre_args      -p1
 patchfiles          patch-doc-example-config-conf.d-10-master.conf.diff \
                     patch-src-master-master-settings.c.diff
 
-checksums           rmd160  37230c065284899fa107ab1c91831ca154f230ac \
-                    sha256  3b61c128f3ef947f5d3e8b1113c321d0597600a3102a8e5b405484a7c049eb51 \
-                    size    3653354
+checksums           core-${version}.tar.gz \
```

Comment:
Is there a reason you added the `distfile` name here? It is not need for a single download and not normally specified. Also, it would be better to simply say `${distfiles}`.

Reply:
no - it's a leftover from when "macports -d install dovecot2" generated checksums for the patches too